fdbserver/KeyValueStoreShardedRocksDB.actor.cpp

```diff
@@ -1725,7 +1770,25 @@ struct ShardedRocksDBKeyValueStore : IKeyValueStore {
 	}
 
 	Future<Optional<Value>> readValue(KeyRef key, IKeyValueStore::ReadType type, Optional<UID> debugID) override {
-		return Optional<Value>();
+		auto shard = shardManager.getDataShard(key);
+		if (shard == nullptr) {
+			TraceEvent(SevWarn, "ShardedRocksDB").detail("Detail", "Read non-exist key range");
```

Comment:
s/SevWarn/SevError.

SS shouldn't have tried to read some keyrange that haven't been added to the KV, or did I missing anything?

Reply:
True for user key space. I think SS will read key space in some cases.

Reply:
Oh, I see! In that case, can we also log the key, e.g., `detail("Key", key)`.